src/components/BaseLayerSwitcher/README.md

```diff
@@ -0,0 +1,39 @@
+#
+
+This demonstrates the use of BaseLayerSwitcher.
+
+```jsx
+import React from 'react';
+import OLMap from 'ol/Map';
+import BaseLayerSwitcher from 'react-spatial/components/BaseLayerSwitcher';
+import BasicMap from 'react-spatial/components/BasicMap';
+import LayerService from 'react-spatial/LayerService';
+import ConfigReader from 'react-spatial/ConfigReader';
+import osmImage from '../../images/baselayer/osm.baselayer.png';
+import osmhotImage from '../../images/baselayer/osm.baselayer.hot.png';
+import travic from '../../images/baselayer/travic.png';
+import openTopoImage from '../../images/baselayer/open.topo.map.png';
+
+const center = [1149722.7037660484, 6618091.313553318];
+const map = new OLMap({ controls: [] });
+const layers = ConfigReader.readConfig(treeData);
+const layerService = new LayerService(layers);
+const layerImages = {
+  osm: osmImage,
+  osmhot: osmhotImage,
+  opentopo: openTopoImage,
+};
```

Comment:
Where is travic used  ?

Reply:
Some residual stuff from when I was testing with geOps tile layers. Removing